### PR TITLE
New conversation

### PR DESCRIPTION
**Pull request by kistitan**
...and some other smaller features

### CODE REVIEW

File: FetLife/fetlife/build.gradle [hunk 1]
```diff
@@ -9,8 +9,8 @@ android {
         applicationId "com.bitlove.fetlife"
         minSdkVersion 15
         targetSdkVersion 22
-        versionCode 600
-        versionName "0.6"
```
Comment (reneklacan) [inline FetLife/fetlife/build.gradle]: you skipped lucky number :D

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/pojos/Conversation.java [hunk 2]
```diff
@@ -104,6 +124,10 @@ public void setDate(long date) {
         this.date = date;
     }
 
+    public boolean isHasNewMessage() {
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/pojos/Conversation.java]: isHas?

**Comment by reneklacan** [comment]
👍

**Comment by jacortinas** [comment]
:+1;

**Pull request merged**